### ISSUE TITLE
Display

### ISSUE THREAD

**Issue by yo2ldk** [tjlab-jf3hzb/Digital_VFO_with_analog_dial#1]
Hello Mr. Uebo,

 I try to add my display based on ILI9488 controller to your excellent project, but unsuccessful..
 Can you please add support for this, when your free time permit that?
  I need it because I want to fit 2.2LCD with my VFO knob,1.8 is a little smaller.

 Thank you in advance, and my best regards !

                                                      de yo2ldk - alexander

                      yo2ldk2 yahoo.com

**Comment by tjlab-jf3hzb** [comment]
ESP-WROOM-32 has 520KB of SRAM, but unfortunately it seems to be usable up to about 100KB as a frame buffer.
Therefore, the upper limit is 160x128 pixels.
If you use ESP32-WROVER-B (with 8MB PSRAM), the limitation will be removed.
However, due to the SPI connection, the data transfer time increases as the number of pixels increases. That causes a problem that the movement of the screen image is not smooth.
I actually made a prototype using an ESP32-WROVER-B and a 240x320 pixel LCD, but I don't think it will be practical.
Because, in addition to the increase in SPI transfer time, the processing time also increased significantly due to slow access speed of PSRAM.
For LCDs larger than 160x128 pixels, it is better to consider using another CPU.
I also made a prototype using PIC32MZ2048. The LCD used is a 240x320 pixel 8bit parallel interface.
This prototype has sufficient performance.
